src/cli/plugin.rs

```diff
@@ -0,0 +1,116 @@
+use crate::serve_session::ServeSession;
+
+use memofs::InMemoryFs;
+use memofs::{Vfs, VfsSnapshot};
+use roblox_install::RobloxStudio;
+use snafu::{ResultExt, Snafu};
+use std::{
+    fs::{self, File},
+    io::{self, BufWriter},
+};
```

Comment:
Can you move the `std` imports to the top of the file and the `crate` imports to below the imports from other crates?

Generally, I like to have imports in three groups:
1. std
2. external crates
3. my own crate

Within those import groups, I let rustfmt sort them.